projects/uswds-components/src/lib/textarea/textarea.component.html

```diff
@@ -0,0 +1,7 @@
+<textarea [ngClass]="{'usa-input--error': showError, 'usa-input--success': !showError && control && control.valid}"
+  [(ngModel)]="model" type="textarea" class="usa-textarea" [attr.id]="name" [attr.placeholder]="placeholder" #textarea
+  [attr.title]="title" [rows]="rowHeight" [minlength]="minlength" [attr.maxlength]="maxlength ? maxlength : 524288"
+  [disabled]="disabled" (ngModelChange)="onInputChange($event)" (focus)="focusChange($event)"
```

Comment:
Because we have `(ngModelChange)` event we're listening for, the `ngModel` attribute does not need `()`. Doing a `[(ngModel)] = model` is the same as doing `[ngModel]="model" (ngModelChange)="model = "$event"`, so we're actually listening to ngModelChange event twice with the current setup. I'm assuming the second one overrides and calls onInputChange, but it's probably safer to just drop the parenthesis from `ngModel` - https://angular.io/guide/two-way-binding#adding-two-way-data-binding

Reply:
Model updated